virttest/env_process.py

```diff
@@ -1038,6 +1038,12 @@ def preprocess(test, params, env):
             params["ker_remove_similar_pci"] = "yes"
         else:
             kernel_extra_params_remove += " pci=nomsi"
+    if params.get("enable_intel_iommu"):
```

Comment:
Not a good design, please also consider if it makes sense for other platforms (e.g. amd ..).

Reply:
@luckyh What I can call to mind is to only enable it on supported platform, or restrict unsupported platform here. Any other suggestion? Thanks.

Reply:
Sounds good. The main point is that it is designed for controling the iommu behavior of guest, so we should point out that it is for **guest**, not intel platform, and I suggest the name `enable_guest_iommu`.

Just forgot one thing, please also add the documentation for it, thanks.

Reply:
@vivianQizhu @luckyh Updated. Please help review again. Thanks.